### DISCUSSION TITLE
Why is there an error with the 'request'?

### DISCUSSION THREAD

**Discussion by HY-Guo1016** [mrdbourke/pytorch-deep-learning#767] [Q&A]
Why is it that when connecting Google Colab to Google's GPU, it runs smoothly, but when connecting to a local environment, it returns an error?
![屏幕截图 2023-12-21 220227](https://github.com/mrdbourke/pytorch-deep-learning/assets/154496898/3fcaad2b-9154-4241-9a90-2e705d9787d8)
![屏幕截图 2023-12-21 220426](https://github.com/mrdbourke/pytorch-deep-learning/assets/154496898/3ea15fa3-1fe8-41ba-a436-009e22e89d91)

**Comment by SabareeshGC** [comment]
You might be behind a proxy or something that is not doing https well or there is a man in the middle attach happening which most of the companies do. You can simply download the file and put it in the folder manually

**Comment by HY-Guo1016** [comment]
Thank you so much!
It did work!

SabareeshGC ***@***.***> 于2023年12月31日周日 01:37写道：
